### PR TITLE
Feature/chart

### PR DESCRIPTION
**Pull request by zhouyuhe**
Jest is now working for reducer

### CODE REVIEW

**Comment by Sharakai** [comment]
`app.test.js` should be called `reducer.test.js` or `reducer.spec.js`

File: src/Reducer.test.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+  UPDATE_CHART_RANGE,
+  UPDATE_KEY_STATS,
+  UPDATE_LATEST_NEWS,
+  UPDATE_COMPANY_OVERVIEW,
+  UPDATE_TOP_PEERS,
+  UPDATE_CHART_DATA,
+  UPDATE_STOCK_TICKER
+} from "./store/constants";
+
+// import { store } from "./store/store";
```
Comment (Sharakai) [inline src/Reducer.test.js]: nit: Get rid of redundant comments.

File: src/Reducer.test.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+// import { Provider } from "react-redux";
+
+// it("renders without crashing", () => {
+//   const div = document.createElement("div");
+//   ReactDOM.render(<Provider store={store}><App /></Provider>, div);
+//   ReactDOM.unmountComponentAtNode(div);
+// });
+
+describe("Stock reducer", () => {
+  let previousState;
```
Comment (Sharakai) [inline src/Reducer.test.js]: I'd definitely look at moving these `previousState` and `newState` variables to within each `describe` closure, just to make sure they're definitely not interacting cross-test.

Comment (steinalex) [reply src/Reducer.test.js]: How would I place the `previousState` within each describe block? My previous state is an object containing the same elements as my `reducer`, so I thought that would need to be accessed by all describe blocks

**Pull request merged**